src/main/java/net/fabricmc/tinyremapper/api/Classpath.java

```diff
@@ -0,0 +1,10 @@
+package net.fabricmc.tinyremapper.api;
+
+public interface Classpath {
```

Comment:
The name of this class is a bit weird considering what it does. I call this Environment in Matcher, so maybe `TrEnvironment`? getByName would have to be called getClass or similar then.